src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -144,7 +145,7 @@ private void InitializeOpenBuffers(SnapshotSpan triggerSpan)
 
                 foreach (var buffer in openBuffers)
                 {
-                    CreateOpenTextBufferManagerForBuffer(buffer, buffer.AsTextContainer().GetRelatedDocuments());
+                    TryPopulateOpenTextBufferManagerForBuffer(buffer, buffer.AsTextContainer().GetRelatedDocuments());
```

Comment:
Yep, you're right, that should be fine. I just wanted to make sure it's been thought through before we have do a follow-up fix here.